### PR TITLE
added requirements for uppmax cluster miarka

### PR DESCRIPTION
**Pull request by b97pla**
---
name: Uppmax Miarka Config
about: A new cluster config
---

Please follow these steps before submitting your PR:

- [ ] If your PR is a work in progress, include `[WIP]` in its title
- [X] Your PR targets the `master` branch
- [ ] You've included links to relevant issues, if any

Steps for adding a new config profile:

- [ ] Add your custom config file to the `conf/` directory
- [ ] Add your documentation file to the `docs/` directory
- [ ] Add your custom profile to the `nfcore_custom.config` file in the top-level directory
- [ ] Add your custom profile to the `README.md` file in the top-level directory
- [ ] Add your profile name to the `profile:` scope in `.github/workflows/main.yml`

### CODE REVIEW

File: conf/uppmax.config [hunk 1]
```diff
@@ -77,6 +91,15 @@ if (hostname.startsWith("i")) {
   params.config_profile_description = 'UPPMAX (Irma) cluster profile provided by nf-core/configs.'
 }
 
+// Cluster: Miarka
+if (hostname.startsWith("m")) {
+  params.max_memory = 357.GB
+  // but what about the 96-core node?
```
Comment (ewels) [inline conf/uppmax.config:97]: I can tell you what - Nextflow will never request it 😅

File: conf/uppmax.config [hunk 2]
```diff
@@ -77,6 +91,15 @@ if (hostname.startsWith("i")) {
   params.config_profile_description = 'UPPMAX (Irma) cluster profile provided by nf-core/configs.'
```
Comment (ewels) [inline conf/uppmax.config:91]: Should we also remove the Irma config?

**Review by ewels — APPROVED** [review]
Looking good 👍🏻

File: conf/uppmax.config (hunk 1 above)
Comment (b97pla) [reply conf/uppmax.config:97]: 👌

File: conf/uppmax.config (hunk 2 above)
Comment (b97pla) [reply conf/uppmax.config:91]: that would make sense but perhaps leave it in at this moment since the cluster is still available for a week or so 😄

**Comment by b97pla** [comment]
thanks for the review! I'm good so please merge when possible 👍

File: conf/uppmax.config (hunk 2 above)
Comment (ewels) [reply conf/uppmax.config:91]: ok!

**Pull request merged**